docs/nfs-utils.md

```diff
@@ -0,0 +1,19 @@
+# Photon NFS utils
+
+To mount network file in Photon
```

Comment:
Mention that its installed by default in FULL installation of Photon and Minimial use tdnf (mention complete command to install nfs on minimal) command to install it.